monasca-agent-base/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.5
+FROM monasca/python:2
```

Comment:
Maybe it would be better to use the full image tag here (with date and timestamps)? That way we'll never need to worry about local caches using an old python-base with a new upstream git. The pr-bot will (soon™) handle updates to the base image automatically.